commcare_connect/program/migrations/0002_update_delivery_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.5 on 2025-11-05 16:17
+
+from django.db import migrations
+
+
+def update_delivery_type(apps, schema_editor):
+    Program = apps.get_model('program', 'Program')
+    ManagedOpportunity = apps.get_model('program', 'ManagedOpportunity')
+
+    for program in Program.objects.all():
+        opps = ManagedOpportunity.objects.filter(program=program)
+
+        for opp in opps:
```

Comment:
Super nit (especially since this is a migration and will run once):
One could probably do something like the following to avoid a python loop:
```
Opportunity.objects.filter(id__in=opps).update(delivery_type=program.delivery_type)
```